README.md

```diff
@@ -9,31 +9,41 @@ JWST Calibration Pipeline
 
 Note
 ----
-Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
+Beginning with version 0.9.0, **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest stable version of the library, we recommend using [conda](https://conda.io/docs/index.html)
+and the [astroconda](https://astroconda.readthedocs.io) channel to install the
+software and using an conda environemnt for the installation:
 
-    % cd jwst
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
```

Comment:
**_Danger_**
Adding the `astroconda-dev` channel to your general conda configuration will make your `astroconda` channel experience very unpleasant. Development packages will supersede stable packages from that point forward.